# Copyright 2016 Preferred Networks, Inc.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
---
sensor_msgs/PointCloud2 fx8_pcl_data
sensor_msgs/Image fx8_x_img
sensor_msgs/Image fx8_y_img
sensor_msgs/Image fx8_z_img
sensor_msgs/PointCloud2 realsense_pcl_data
sensor_msgs/Image realsense_rgb_img
sensor_msgs/Image realsense_x_img
sensor_msgs/Image realsense_y_img
sensor_msgs/Image realsense_z_img
